devkit/e2e/src/start.sh

```diff
@@ -13,15 +13,15 @@ DIR_SANDBOX=$(realpath $DIR/../../sandbox/src)
 source $DIR_SANDBOX/common.sh
 
 # If there's a fatal error or users Ctrl+C it will tear down setup
-trap 'stop' SIGINT SIGKILL SIGTERM ERR EXIT
+trap 'stop; exit 1' SIGINT SIGKILL SIGTERM ERR EXIT
 
 # start sandbox
 start
 
 # catch error if command fails
 set -Eeuo pipefail
 # Run tests with env variables
-npx jest --ci
+npx jest --ci --coverage || exit 1
```

Comment:
why do we need coverage?

Reply:
Actually not needed! Thanks for the catch